### PR TITLE
chore(cf-style-const): rename gradient keys in theme

### PR DESCRIPTION
**Pull request by manatarms**
BREAKING CHANGE: This change will break all places that use gradients from theme

### CODE REVIEW

**Comment by manatarms** [comment]
Places gradients are used right now.

In www-next
1) overview page - CardHeader

In cf-ux
1) OnboardingContentView
2) OnboardingView
3) cf-component-presentation -> TopGradient

File: packages/cf-builder-pagination/test/__snapshots__/PaginationBuilder.js.snap [hunk 1]
```diff
@@ -26,7 +26,7 @@ exports[`should render pagination 1`] = `
       </a>
     </li>
     <li
-      className="m p ad ae   s af ag"
```
Comment (manatarms) [inline packages/cf-builder-pagination/test/__snapshots__/PaginationBuilder.js.snap]: Fela changed the spaces on the snapshots for some reason. Not sure why.

**Comment by koddsson** [comment]
Did publishing with the `chore` keyword work alright?

**Comment by manatarms** [comment]
@koddsson It was beautiful. It just figured out everything that needed a bump, which in this case was `cf-style-const`. So nice! Thanks for this.

**Pull request merged**